src/core/components/UserInfo.js

```diff
@@ -1,8 +1,35 @@
-export default function UserInfo() {
+import { useCallback, useEffect } from 'react';
+import { connect } from 'react-redux';
+import UserAPIClient from '../services/UserAPIClient';
+
+const Component = function UserInfo(props) {
+   const getInfoPlayer = useCallback(async () => {
```

Comment:
Because this action is only done once when loading into the website, if we do not use useCallback, the website will still send requests when the state changes.